.travis.yml

```diff
@@ -236,10 +236,10 @@ jobs:
 
 install:
   - ccache --max-size=1G
-  - make -C src minisat2-download
+  - make -C src glucose-download
   - make -C src/ansi-c library_check
-  - make -C src "CXX=${COMPILER}" "CXXFLAGS=-Wall -Werror -pedantic -O2 -g ${EXTRA_CXXFLAGS}" -j2
-  - make -C src "CXX=${COMPILER}" "CXXFLAGS=-Wall -Werror -pedantic -O2 -g ${EXTRA_CXXFLAGS}" -j2 clobber.dir memory-models.dir
```

Comment:
I don't think that's a good idea to remove these flags

Reply:
Sure, it is gone.